common/src/main/java/com/google/auto/common/MoreTypes.java

```diff
@@ -819,11 +819,6 @@ public Boolean visitNoType(NoType noType, Void p) {
       return noType.getKind().equals(TypeKind.VOID);
     }
 
-    @Override
-    public Boolean visitPrimitive(PrimitiveType type, Void p) {
```

Comment:
I think it might have made more sense for it to be like this from the start, but I am concerned that changing it now could break code depending on the previous behaviour. I would prefer to change the spec of `isType` to make it clearer exactly what it means when it says "a type that can be referenced by a `Class`". Primitives _can_ be referenced by a `Class`, for example `int.class`.

Reply:
The change has been reverted.